frontend-ssr/pages/searchExerciseSubmissions.tsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @typescript-eslint/no-empty-interface */
```

Comment:
This should be a generic component. We're gonna re-use it in multiple pages so it shouldn't what we're searching.